Documentation/network/servicemesh/istio.rst

```diff
@@ -36,35 +36,54 @@ You can confirm the result with the following command:
     $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep bpf-lb-sock-hostns
     bpf-lb-sock-hostns-only: "true"
 
+
+Istio uses a CNI plugin to implement functionality for both sidecar and ambient modes.
+To ensure that Cilium does not interfere with other CNI plugins on the node, it is important to set the ``cni-exclusive``
+parameter in the Cilium ConfigMap to ``false``. This can be achieved by using the ``--set`` flag with the ``cni.exclusive``
+Helm value set to ``false``.
+You can confirm the result with the following command:
+
+.. code-block:: shell-session
+
+    $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep cni-exclusive
+    cni-exclusive: "false"
+
 .. _gsg_istio_cnp:
 
 Istio configuration
 ===============================
 
-When you deploy Istio, be aware of:
+When you deploy Cilium and Istio together, be aware of:
+
+* Either Cilium or Istio L7 HTTP policy controls can be used, but it is not recommended to use **both** Cilium and Istio
+  L7 HTTP policy controls at the same time, to avoid split-brain problems.
+
+  In order to use Cilium L7 HTTP policy controls (for example, :ref:`l7_policy`) with Istio (sidecar or ambient modes), you must disable Istio mTLS by configuring
+  ``mtls.mode=DISABLE`` under Istio's `PeerAuthentication <https://istio.io/latest/docs/reference/config/security/peer_authentication/#PeerAuthentication>`_,
+  as otherwise the traffic between Istio-managed workloads will be encrypted by Istio with mTLS, and not accessible to Cilium for the purposes of L7 policy enforcement.
 
-* The new experimental `ambient mesh <https://istio.io/latest/blog/2022/introducing-ambient-mesh/>`_ 
-  data plane is not supported, as it interferes with the Cilium data plane.
+  If using Istio L7 HTTP policy controls, policy will be managed in Istio and disabling mTLS between workloads is not required.
 
-* mTLS mode ``STRICT`` or ``PERMISSIVE`` (default) are not compatible with Cilium HTTP network policy. 
-  To use an HTTP-based network policy (for example, :ref:`l7_policy`), 
-  you must configure ``mtls.mode=DISABLE`` under Istio's ``PeerAuthentication``.
+* If using Istio mTLS in ambient mode, traffic between ambient workloads will be
+  `encrypted and tunneled in and out of the pods by Istio over port 15008 <https://istio.io/latest/docs/ops/ambient/usage/traffic-redirection/>`_.
+  This means that any egress NetworkPolicies for workloads that assume other ports will need to be updated accordingly, and that Istio must be used
+  to enforce policy for traffic between Istio-managed workloads, as Cilium will have no visibility into tunneled and encrypted traffic entering and leaving the pod.
```

Comment:
> Istio must be used to enforce policy for traffic between Istio-managed workloads

can we make it a bit more concrete, e.g. ` Istio can be used to enforce policy for traffic between Istio-managed workloads at L4 and above`

Reply:
Reworded the section again, agree.